test/integration/targets/cloudscale_server/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
+  - prepare_tests
```

Comment:
@resmo I don't know why this role does not exist, but this dependency is listed in all test roles unter integration/targets so I thought this should be added by default.